test/support/mocks.ex

```diff
@@ -0,0 +1,5 @@
+# This file houses definitions for defining Mox mocks.
+
+Mox.defmock(Redis.Mock, for: Dotcom.Redis.Behaviour)
+Mox.defmock(Redix.Mock, for: Dotcom.Redix.Behaviour)
+Mox.defmock(Redix.PubSub.Mock, for: Dotcom.Redix.PubSub.Behaviour)
```

Comment:
If the behaviours are only used for testing, should they be defined in a testing folder?

Reply:
Behaviours can be used for other purposes such as stubs or even custom implementations so it makes sense to keep them in lib. A good example would be a behaviour of the V3 API that we then implement ourselves and also mock.

Reply:
See this as an example: https://github.com/mbta/my_charlie/tree/main/lib/my_charlie/reduced_fares_applications/steps/identity_check/api